db/migrate/20150115155947_add_scaptimony_scap_content_digest.rb

```diff
@@ -0,0 +1,11 @@
+class AddScaptimonyScapContentDigest < ActiveRecord::Migration
+  def change
+    add_column :scaptimony_scap_contents, :digest, :string, :limit => 128
+    Scaptimony::ScapContent.find_each do |content|
```

Comment:
this can cause problems in future if we add more attributes and validations to to ScapContent, you should create some fake class for this

e.g.
```ruby
class Scaptimony::FakeScapContent << ActiveRecord::Base
  self.table_name = 'scaptimony_scap_contents'

  def digest
    self[:digest] ||= Digest::SHA256.hexdigest "#{scap_file}"
  end
end
```

and then use this class to iterate over all contents